crates/prover/src/core/backend/simd/circle.rs

```diff
@@ -275,6 +276,7 @@ impl PolyOps for SimdBackend {
         )
     }
 
+    // TODO(spapini): Parallelize.
```

Comment:
Hey! I saw this TODO and addressed it. I used the `SendMutU32Ptr` to share pointers between threads :)
[](https://github.com/starkware-libs/stwo/pull/703)